packages/forms-web-app/src/controllers/lpa-dashboard/add-remove-users.js

```diff
@@ -18,10 +21,14 @@ const getAddRemoveUsers = async (req, res) => {
 		delete req.session.removeUserEmailAddress;
 	}
 
+	const user = getLPAUserFromSession(req);
+	const users = await getUsers(user.lpaCode);
```

Comment:
wonder if usersList or similar might be better, for clarity and to avoid any confusion between user/users